### ISSUE TITLE
The program only runs as root

### ISSUE THREAD

**Issue by Kernitskiy** [libredeb/comice-about#4]
Hello!  The program is super, but after installation it does not start for me.  Only by going into the folder with programs from under root can you run it.  How to fix it?

**Comment by libredeb** [comment]
> Hello! The program is super, but after installation it does not start for me. Only by going into the folder with programs from under root can you run it. How to fix it?

Yes, this behavior is due to poor documentation... I just fixed several bugs that I found. Could you try to follow the new instructions again and let me know if this behavior persists?

**Comment by libredeb** [comment]
Issue closed due to lack of confirmation and inactivity
